database/boltdb.go

```diff
@@ -28,62 +28,26 @@ import (
 	"github.com/golang/glog"
 )
 
-//DbTable interface that needs to be supported
-//for the table to be handled by the database
-type DbTable interface {
-	// Creates the backing map
-	NewTable()
-
-	// Name of the table as stored in the database
-	Name() string
-
-	// Allocates and returns a single value in the table
-	NewElement() interface{}
-
-	// Add an value to the in memory table
-	Add(k string, v interface{}) error
-}
-
-// A TableDBProvider represents a persistent database provider
-// that can be used to store map, arrays or any other type of
-// tables into a database
-type TableDBProvider interface {
-	//Initializes the Database
-	DbInit(dir string, file string) error
-	//Populates the in-memory table from the database
-	DbTableRebuild(table DbTable) error
-	//Closes the database
-	DbClose() error
-	//Creates the tables if the tables do not already exist in the database
-	DbTableInit(tables []string) error
-	//Adds the key value pair to the table
-	DbAdd(table string, key string, value interface{}) error
-	//Adds the key value pair to the table
-	DbDelete(table string, key string) error
-	//Retrives the value corresponding to the key from the table
-	DbGet(table string, key string) (interface{}, error)
-}
-
-type tableBoltDB struct {
+type boltDB struct {
 	Name string
 	DB   *bolt.DB
 }
 
-func newTableBoltDb() *tableBoltDB {
-	return &tableBoltDB{
-		Name: "tableBolt.DB",
+type dbProvider boltDB
```

Comment:
Good point, removing that, mybe it was just a copy and paste issue from networking code: https://github.com/01org/ciao/blob/master/networking/libsnnet/database.go#L78